pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix

```diff
@@ -343,7 +343,12 @@ in rec {
         mv $NIX_BUILD_TOP/temp "$PWD/deps/${package.name}"
         cd $PWD
 
-        ln -s ${deps}/deps/${package.name}/node_modules "deps/${package.name}/node_modules"
+        # TODO: Use
```

Comment:
@happysalada As much as I'd like to get this merged, this is still a remaining issue that will break other packages using yarn2nix.

Reply:
Couldnt that step from yarn2nix be overriden ?

Thanks for bringing this to my attention.

The other solution would be to add an option in yarn2nix that defaults to not removing this.

Reply:
Could be. I might find time tomorrow to look into a solution that would be mergable.

Reply:
I think I found a workaround for this issue, which allowed me to remove the change from `mkYarnPackage`. I have no idea how this works though.